client/src/components/backend/Event/ListItem.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent, PropTypes } from 'react';
+import { Event } from 'components/frontend';
+
+export default class EventListItem extends PureComponent {
+
+  static displayName = "Event.ListItem";
+
+  static propTypes = {
+    event: PropTypes.object,
+    destroyHandler: PropTypes.func
+  };
+
+  constructor(props) {
+    super(props);
+    this.triggerDestroy = this.triggerDestroy.bind(this);
+  }
+
+  triggerDestroy(event) {
+    event.preventDefault();
+    this.props.destroyHandler(this.props.entity);
+  }
+
+  render() {
+    const event = this.props.entity;
+    if (!event) return null;
+    const attr = event.attributes;
+    const content = attr.excerpt ? attr.excerpt : attr.subjectTitle;
+    return (
+      <li className="list-item">
```

Comment:
You need a unique key on this <li> (which I've added). Remember, any time you're rendering a collection like this in React, the wrapper element on each item needs a key.